src/app/services/chart.service.ts

```diff
@@ -13,8 +13,23 @@ import * as _ from 'lodash';
 @Injectable()
 export class ChartService {
 
+    private multiChartScrubberInfo = new Subject();
```

Comment:
A subject seems pretty correct for this implementation. While I have limited experience as well, Subjects can be frowned upon as they have state, which is no good for the functional programming paradigm, and so see limited use.

I found this, appears to be a great deep dive into when to and when to not use Subjects:
http://davesexton.com/blog/post/To-Use-Subject-Or-Not-To-Use-Subject.aspx